plugins/GSdx/GSWnd.h

```diff
@@ -60,19 +60,20 @@ class GSWndGL : public GSWnd
 {
 protected:
 	bool m_ctx_attached;
-	int m_vsync;
+	std::atomic_bool m_vsync_change_requested;
```

Comment:
Why not use `std::atomic<bool>` ?

Reply:
@ssakash yeah I checked google before asking the reason ? I don't know why they add it, maybe for raw C purpose.